src/main/java/org/folio/edge/controller/exception/V2ExceptionHandlerController.java

```diff
@@ -0,0 +1,72 @@
+package org.folio.edge.controller.exception;
+
+import java.util.stream.Collectors;
+
+import org.folio.edge.domain.exception.CirculationProcessException;
+import org.folio.edge.domain.exception.EntityNotFoundException;
+import org.folio.edge.dto.InnReachResponseDTO;
+import org.folio.edge.mapper.InnReachErrorMapper;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+@Log4j2
+@RequiredArgsConstructor
+@RestControllerAdvice(basePackages = "org.folio.edge.controller")
```

Comment:
All the controllers are placed inside "org.folio.edge.controller", so this one will override existing error handler which is being used for authentication flow.

Reply:
Done